src/main/drivers/serial_softserial.c

```diff
@@ -136,10 +210,9 @@ static void serialTimerTxConfig(const timerHardware_t *timerHardwarePtr, uint8_t
         }
     } while (isTimerPeriodTooLarge(timerPeriod));
 
-    uint8_t mhz = SystemCoreClock / 1000000;
+    uint8_t mhz = baseClock / 1000000;
```

Comment:
You mean to fix `timerConfigure` and `configTimeBase`?

May be... There are only half a dozen callers... I don't know if it worth the effort :wondering: